modules/scala-3-step-filter/src/main/scala/ch/epfl/scala/debugadapter/internal/stepfilter/ScalaStepFilterBridge.scala

```diff
@@ -30,72 +31,80 @@ class ScalaStepFilterBridge(
   private def throwOrWarn(msg: String): Unit = {}
 //    if (testMode) throw new Exception(msg)
   //  else warn(msg)
-  def convertTypeToString(isArgs: Boolean, t: Type): String = {
+  def formatType(isArgs: Boolean, t: Type): String = {
```

Comment:
Do you still need `isArgs`?
```suggestion
  def formatType(t: Type): String = {
```